pkg/api/customization/cred/store.go

```diff
@@ -5,14 +5,16 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/labels"
+
```

Comment:
eh?

Reply:
Separating the k8s imports from github imports, I've noticed some people prefer this so I am conforming.

Reply:
well if the cool kids are doing it